### DISCUSSION TITLE
How we can add billing details in GitHub? Just for achievement

### DISCUSSION THREAD

**Discussion by TheTeamAlexa** [TheTeamAlexa/TheTeamAlexa#2] [Q&A] [answered]
How we can add billing details in GitHub?

**Comment by Team-Alisha** [comment, marked as answer]
To add billing details in GitHub, you can follow these steps:

Go to the "Billing & plans" section of your GitHub account. You can access it by clicking on your profile picture in the top right corner of the screen, and then selecting "Settings" from the dropdown menu.

Click on the "Billing & plans" tab, and then select the "Payment method" option from the left-hand menu.

Click on the "Add a payment method" button, and then select your preferred payment method from the options provided. GitHub accepts credit/debit cards and PayPal.

Enter your billing details, including your name, billing address, and payment information.

Click on the "Add payment method" button to save your details.

Once you have added your billing details, you can select a plan for your GitHub account or upgrade to a higher plan. You can also view your billing history and update your billing details at any time by going to the "Billing & plans" section of your GitHub account.
